### PR TITLE
Use better red.

### LINKED ISSUE #229: Update the reds (colors!)
Some of the red shades are `#f00` and don't have a sufficient color contrast ratio with the white background. As already done in other parts, see for example https://github.com/Yoast/wordpress-seo/issues/4058, we should standardise on the new WordPress red `#dc3232` for white backgrounds.

screen shot 2016-07-05 at 16 11 27

About this specific error message, maybe also toggle a CSS class instead of using inline styles.

### PR DESCRIPTION
**Pull request by afercia**
Fixes #229.
Also, some coding standards.

### CODE REVIEW

File: classes/class-wpseo-news.php [hunk 1]
```diff
@@ -243,7 +243,7 @@ public function enqueue_edit_post() {
 	 * Print the edit post CSS
 	 */
 	public function edit_post_css() {
-		echo "<style type='text/css'>.wpseo-news-input-error{border:1px solid #ff0000 !important;}</style>" . PHP_EOL;
+		echo "<style type='text/css'>.wpseo-news-input-error{border:1px solid #dc3232 !important;}.wpseo-news-error-message{color:#dc3232;font-weight:bold;}</style>" . PHP_EOL;
```
Comment (terw-dan) [inline classes/class-wpseo-news.php]: Can't we do without the !important here?

**Comment by terw-dan** [comment]
CR done, only 1 small question @afercia

**Comment by afercia** [comment]
Alternate approach. Was thinking we could use the native WordPress CSS classes `form-invalid` and `error-message` so we could also remove the inline stylesheet. However, these WordPress CSS classes might be removed in the future (?) and expose us to maintenance issues. Thoughts?
Otherwise, we can simply keep the inline style but we should use a selector with a higher specificity or `!important`.

**Comment by afercia** [comment]
Forgot to say that the WordPress `error-message` uses a `#f00` color but that should be fixed upstream and there's a ticket for that: https://core.trac.wordpress.org/ticket/35622

**Comment by atimmer** [comment]
CR Done 👍🏻

**Comment by atimmer** [comment]
Acceptance done 👍🏻, @afercia I got some merge conflicts, can you fix them so I can merge this?

**Comment by afercia** [comment]
Merged conflicts cc @atimmer

**Pull request merged**